src/main/java/edu/kit/ipd/crowdcontrol/objectservice/rest/RequestUtil.java

```diff
@@ -40,6 +44,15 @@ public static boolean getQueryBool(Request request, String param, boolean defaul
             return defaultValue;
         }
 
-        return Boolean.parseBoolean(paramValue);
+        switch(paramValue.toLowerCase()) {
+            case "1":
+            case "true":
+                return true;
+            case "0":
+            case "false":
+                return false;
+            default:
+                throw new BadRequestException("Query parameter '%s' must be a valid boolean, '%s' given. Use true / 1 / false / 0.", param, paramValue);
```

Comment:
I think it's harder to get that there are only two values, but I don't really care. Changed.